docs/helpshift-widget/api.mdx

```diff
@@ -0,0 +1,433 @@
+---
+sidebar_position: 20
+title: Helpshift APIs
+description: Learn more about Helpshift APIs
+---
+
+import {
+  Admonition,
+  CodeBlock,
+  Tabs,
+  TabItem,
+  LatestSdkVersion,
+  Centered,
+  Image,
+  Intro,
+  SideBySide,
+  DownloadButton,
+  Steps,
+  Step,
+} from "@site/src/components/forDocs";
+
+# Helpshift APIs
+
+<Intro>
+
+</Intro>
+
+## Open Widget {#open-widget}
+
+If you want to open the helpshift widget window at any time (apart from an explicit user click on widget launcher button), you can do it by calling the following API. This could be useful for triggering proactive help.
```

Comment:
Let's use `Helpshift Widget` instead of helpshift widget

Reply:
Done